micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/SideBar/CitizenSideBar.js

```
@@ -48,117 +56,153 @@ const Profile = ({ info, stateName, t }) => {
+  const [selectedCity, setSelectedCity] = useState([]); //selectedCities?.[0]?.value
+  const [selected, setselected] = useState(selectedLanguage);
+  let selectedCities = [];
+  const { isLoading, data } = Digit.Hooks.useAccessControl();
+  const tenantId = Digit.ULBService.getCurrentTenantId();
   const { t } = useTranslation();
   const history = useHistory();
+
+  const stringReplaceAll = (str = "", searcher = "", replaceWith = "") => {
+    if (searcher == "") return str;
+    while (str?.includes(searcher)) {
+      str = str?.replace(searcher, replaceWith);
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)